csharp/tutorials/AsyncStreams/finished/IssuePRreport/IssuePRreport/Program.cs

```diff
@@ -60,8 +60,12 @@ static async Task Main(string[] args)
             };
 
             // <SnippetEnumerateAsyncStream>
+            int num = 0;
             await foreach (var issue in runPagedQueryAsync(client, PagedIssueQuery, "docs"))
+            {
                 Console.WriteLine(issue);
+                Console.WriteLine($"Received {++num} issues in total");
```

Comment:
do you really want to print this in the middle of the loop?

Reply:
Yes, because that was the question on https://github.com/dotnet/docs/issues/11522

I felt the same way you did when I wrote the sample. But, I think showing how async enumerables means you don't need the IProgress interface to track progress is enlightening.